include/mgp.hpp

```diff
@@ -1543,6 +1545,8 @@ inline Type ToAPIType(mgp_value_type type) {
 
 // Id:
 
+inline constexpr Value::StealType steal{};
```

Comment:
I am not sure if this was an idea of how to add a `steal` object?

Reply:
I would say just put it into the `mgp` namespace as it might be useful for other types also, not just for `Value`.